### PR TITLE
Fix tensorflow failing

### LINKED ISSUE #17102: Test failure in Travis
```console
sympy/printing/tests/test_tableform.py[2] ..                                [OK]
WARNING: Logging before flag parsing goes to stderr.
W0625 22:05:27.962558 139676919949120 deprecation_wrapper.py:119] From /home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.5.dev0-py2.7.egg/sympy/printing/tests/test_tensorflow.py:28: The name tf.Session is deprecated. Please use tf.compat.v1.Session instead.
2019-06-25 22:05:27.964012: I tensorflow/core/platform/cpu_feature_guard.cc:145] This TensorFlow binary is optimized with Intel(R) MKL-DNN to use the following CPU instructions in performance critical operations:  SSE4.1 SSE4.2 AVX AVX2 FMA
To enable them in non-MKL-DNN operations, rebuild TensorFlow with the appropriate compiler flags.
2019-06-25 22:05:27.971795: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2300000000 Hz
2019-06-25 22:05:27.972028: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x55a1047c3460 executing computations on platform Host. Devices:
2019-06-25 22:05:27.972094: I tensorflow/compiler/xla/service/service.cc:175]   StreamExecutor device (0): <undefined>, <undefined>
OMP: Info #212: KMP_AFFINITY: decoding x2APIC ids.
OMP: Info #210: KMP_AFFINITY: Affinity capable, using global cpuid leaf 11 info
OMP: Info #154: KMP_AFFINITY: Initial OS proc set respected: 0,1
OMP: Info #156: KMP_AFFINITY: 2 available OS procs
OMP: Info #157: KMP_AFFINITY: Uniform topology
OMP: Info #179: KMP_AFFINITY: 1 packages x 1 cores/pkg x 2 threads/core (1 total cores)
OMP: Info #214: KMP_AFFINITY: OS proc to physical thread map:
OMP: Info #171: KMP_AFFINITY: OS proc 0 maps to package 0 thread 0 
OMP: Info #171: KMP_AFFINITY: OS proc 1 maps to package 0 thread 1 
OMP: Info #250: KMP_AFFINITY: pid 16149 tid 16149 thread 0 bound to OS proc set 0
2019-06-25 22:05:27.972641: I tensorflow/core/common_runtime/process_util.cc:115] Creating new thread pool with default inter op setting: 2. Tune using inter_op_parallelism_threads for best performance.
sympy/printing/tests/test_tensorflow.py[5] E2019-06-25 22:05:27.992492: W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
....                          [FAIL]
sympy/printing/tests/test_theanocode.py[38] ................/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/theano/gof/opt.py:251: UserWarning: WARNING: Supervisor is not added. Please build a FunctionGraphvia theano.compile.function_module.std_graph()or add the Supervisor class manually.
  sub_prof = optimizer.optimize(fgraph)
....................
..                                                                          [OK]
sympy/printing/tests/test_tree.py[1] .                                      [OK]
sympy/vector/tests/test_printing.py[4] ....                                 [OK]
________________________________ slowest tests _________________________________
test_issue_15716 - Took 10.976 seconds
________________________________________________________________________________
________ sympy/printing/tests/test_tensorflow.py:test_tensorflow_matrix ________
Traceback (most recent call last):
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.5.dev0-py2.7.egg/sympy/utilities/runtests.py", line 1308, in _timeout
    function()
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.5.dev0-py2.7.egg/sympy/printing/tests/test_tensorflow.py", line 55, in test_tensorflow_matrix
    _compare_tensorflow_matrix((M,), expr)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.5.dev0-py2.7.egg/sympy/printing/tests/test_tensorflow.py", line 32, in _compare_tensorflow_matrix
    f = lambdify(variables, expr, 'tensorflow')
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.5.dev0-py2.7.egg/sympy/utilities/lambdify.py", line 695, in lambdify
    buf = _get_namespace(m)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.5.dev0-py2.7.egg/sympy/utilities/lambdify.py", line 811, in _get_namespace
    _import(m)
  File "/home/travis/miniconda/envs/test-environment/lib/python2.7/site-packages/sympy-1.5.dev0-py2.7.egg/sympy/utilities/lambdify.py", line 159, in _import
    namespace[sympyname] = namespace[translation]
KeyError: 'log'

```

### PR DESCRIPTION
**Pull request by sylee957**
#### References to other Issues or PRs

Fixes #17102

#### Brief description of what is fixed or changed

I have found the reason why and this may be an workaround
https://github.com/sympy/sympy/issues/17102#issuecomment-505761332
See if it passes

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/utilities/lambdify.py [hunk 1]
```diff
@@ -100,7 +100,7 @@
     "mpmath": (MPMATH, MPMATH_DEFAULT, MPMATH_TRANSLATIONS, ("from mpmath import *",)),
     "numpy": (NUMPY, NUMPY_DEFAULT, NUMPY_TRANSLATIONS, ("import numpy; from numpy import *; from numpy.linalg import *",)),
     "scipy": (SCIPY, SCIPY_DEFAULT, SCIPY_TRANSLATIONS, ("import numpy; import scipy; from scipy import *; from scipy.special import *",)),
-    "tensorflow": (TENSORFLOW, TENSORFLOW_DEFAULT, TENSORFLOW_TRANSLATIONS, ("import_module('tensorflow')",)),
+    "tensorflow": (TENSORFLOW, TENSORFLOW_DEFAULT, TENSORFLOW_TRANSLATIONS, ("from tensorflow import *",)),
```
Comment (czgdp1807) [inline sympy/utilities/lambdify.py]: Ah! I see that was the fix, the `tensorflow` attributes weren't being imported, right? Well, what about the one time warning? Was that also due to the import issues?

**Comment by oscargus** [comment]
Looking promising! Merge once the tests finish.

**Comment by czgdp1807** [comment]
The warning is still thrown during the build,
```
W tensorflow/compiler/jit/mark_for_compilation_pass.cc:1412] (One-time warning): Not using XLA:CPU for cluster because envvar TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set.  If you want XLA:CPU, either set that envvar, or use experimental_jit_scope to enable XLA:CPU.  To confirm that XLA is active, pass --vmodule=xla_compilation_cache=1 (as a proper command-line flag, not via TF_XLA_FLAGS) or set the envvar XLA_FLAGS=--xla_hlo_profile.
```
I tried to fix it in https://github.com/sympy/sympy/pull/17103. Is it worthy enough to re-open that PR because its just a warning but still we should avoid it.

**Comment by sylee957** [comment]
@czgdp1807 
I'm more worried about the deprecation warnings in #17105 though,
but you can go opening up a PR if it helps making the log less verbose.

**Comment by jksuom** [comment]
The title of the [commit](https://github.com/tensorflow/tensorflow/commit/ba994787e69ade025949c79fc95932e83ac0a1af#diff-73244848675e057e66040c0bcef78b51) introducing the warning is "Warn if autojit is enabled but CPU autojit is disabled". So it looks like either autojit should be disabled of CPU autojit enabled.

**Comment by jksuom** [comment]
I think that the deprecation warnings prepare for tensorflow 2. So they should be handled before that is released.

**Comment by czgdp1807** [comment]
> Warn if autojit is enabled but CPU autojit is disabled

So, I think the environment variable setting `TF_XLA_FLAGS=--tf_xla_cpu_global_jit was not set` should be added in travis.yml.



> I think that the deprecation warnings prepare for tensorflow 2. So they should be handled before that is released.

As far as I can figure out from the warning, I believe, we should change, from `tf.Session` -> `tf.compat.v1.Session`, then I think I can fix it. I am re-opening the PR #17103 .

**Comment by sylee957** [comment]
Besides, I see the tensorflow developers had fixed https://github.com/tensorflow/tensorflow/issues/30184 to https://github.com/tensorflow/tensorflow/pull/30236. So this was really an issue upstream, and if we find any advantage of using `tensorflow.__dict__` for further development later, we can.

**Pull request merged**